### DISCUSSION TITLE
Fabricating assault rifle magazine doesn't consume all materials

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#13909] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Fixed here https://github.com/FakeFishGames/Barotrauma/issues/13346 however the issue still seems to occur:
```
<Fabricate suitablefabricators="fabricator" requiredtime="35" requiresrecipe="true">
  <RequiredSkill identifier="weapons" level="40" />
  <RequiredItem identifier="magnesium" />
  <RequiredItem tag="advmunition_tip" description="fabricationdescription.advmunition_tip" />
  <RequiredItem tag="advmunition_core" description="fabricationdescription.advmunition_core" />
  <RequiredItem tag="advmunition_jacket" description="fabricationdescription.advmunition_jacket" />
  <RequiredItem identifier="steel" />
</Fabricate>
```
![slika](https://github.com/FakeFishGames/Barotrauma/assets/115734437/eaa32380-51ce-45ca-93f0-1ab20daccf50)
Had magnesium, titanium, uranium, zinc, steel bar in inventory and fabricator only pulled 4 items:
![slika](https://github.com/FakeFishGames/Barotrauma/assets/115734437/e872d3fa-51c7-4b8a-8d34-245af91a214e)

### Reproduction steps

Fabricate Assault Rifle Magazine with magnesium, 2 titanium, zinc, steel bar or magnesium, titanium, uranium, zinc, steel bar

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/6c7000e4fa0067d3e72c3142ab11b3aee6bbf44a (commit in our private repo)

**Comment by Jasontti** [comment]
Tested the fix and found no issues.
Closing discussion as resolved.
